client/src/views/pages/search.js

```diff
@@ -132,12 +132,21 @@ export default function (props) {
               <>
                 <CardBody className="text-center border-bottom">
                   <h4>Steam Profile</h4>
-                  {data.steamUser.name && data.steamUser.avatarFull && (
+                  {data.steamUser.name && data.steamUser.avatarFull ? (
                     <img
                       alt={`${data.steamUser.name}'s avatar`}
                       src={data.steamUser.avatarFull}
                       className="rounded-circle mb-4"
                     />
+                  ) : (
+                    <img
+                      alt={`${
+                        data.steamUser.name ? data.steamUser.name : data.steamUser.id
```

Comment:
As above, maybe replace the ternary operator with `data.steamUser.name || data.steamUser.id` so it falls back to the latter if the former evaluates to false.